<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Library_windowed</class>
 <widget class="QWidget" name="Library_windowed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Artists</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="lv_artist"/>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Albums</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="lv_album"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tb_title"/>
       </item>
      </layout>
     </item>
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_search"/>
       </item>
       <item>
        <widget class="QPushButton" name="btn_clear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
